services/workflows-service/src/transaction/transaction.controller.external.ts

```diff
@@ -59,7 +59,7 @@ export class TransactionControllerExternal {
   }
 
   @Get()
-  @UseCustomerAuthGuard()
+  // @UseGuards(CustomerAuthGuard)
```

Comment:
Why?

Reply:
There's currently no way of applying both session auth and a token. It was a group decision to address it at a later point. I will create a task.

Reply:
Task created.